independent-projects/tools/devtools-testing/src/main/java/io/quarkus/devtools/testing/RegistryClientTest.java

```diff
@@ -0,0 +1,18 @@
+package io.quarkus.devtools.testing;
+
+import org.junit.jupiter.api.extension.AfterAllCallback;
+import org.junit.jupiter.api.extension.BeforeAllCallback;
+import org.junit.jupiter.api.extension.ExtensionContext;
+
+public class RegistryClientTest implements BeforeAllCallback, AfterAllCallback {
```

Comment:
Now anything that uses this class will be setup to use the quarkus-bom platform.

Reply:
let me try without it to see the result

Reply:
Ok I am not using it anymore from the devtools-testing tests